node/cmd/calico/cni_conf_test.go

```diff
@@ -23,13 +23,17 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var _ = Describe("CNI config template tests", func() {
+var _ = Describe("Windows CNI config template tests", func() {
 	It("should be valid JSON", func() {
-		f, err := ioutil.ReadFile("../../windows-packaging/TigeraCalico/cni.conf.template")
```

Comment:
do we need to move a file in order to make this change?

Reply:
The original path is wrong and doesn't exist.